tests/load/isolation_locustfile.py

```
@@ -0,0 +1,122 @@
+logger = logging.getLogger(__name__)
+
+LOAD_PROFILES = {
+    "smoke": {"users": 5, "spawn_rate": 1, "run_time": "2m"},
+    "thorough": {"users": 10, "spawn_rate": 1, "run_time": "5m"},
+}
+
+_isolation_violations: list[str] = []
+
+
+@events.init.add_listener
+def on_init(environment, **kwargs):  # type: ignore[no-untyped-def]
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `on_init`

Add return type annotation: `None`

(ANN202)